lib/game.rb

```diff
@@ -0,0 +1,75 @@
+require "pry"
+
+class Game
+
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [
+                      [0, 1, 2],
+                      [3, 4, 5],
+                      [6, 7, 8],
+                      [0, 3, 6],
+                      [1, 4, 7],
+                      [2, 5, 8],
+                      [0, 4, 8],
+                      [2, 4, 6]
+  ]
+
+  def initialize(player_1=Players::Human.new("X"), player_2=Players::Human.new("O"), board=Board.new)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+  end
+
+  def turn_count
+    @board.cells.count{|cell| cell == "O" || cell == "X"}
+  end
+
+  def current_player
+    (self.turn_count.even? == true) ? player_1 : player_2
```

Comment:
You can use the implicit return rather than check for boolean equality.
`self.turn_count.even? ? player_1 : player_2`